src/state/hooks/message-preset.js

```diff
@@ -0,0 +1,75 @@
+// @flow
```

Comment:
We ship default english messaging. This costs users some KB, but it means that we push accessibility by default - it is not opt in. This fulfils the libraries goal of being 

> Beautiful, **accessible** drag and drop for lists with React.js